defaults/main.yml

```diff
@@ -74,7 +74,7 @@ wireguard_interface_restart: false
 #######################################
 
 # Set to "false" if package cache should not be updated
-wireguard_ubuntu_update_cache: "true"
+wireguard_update_cache: "true"
```

Comment:
See my README comment about this.

Reply:
`wireguard_ubuntu_update_cache` now defaults to `wireguard_update_cache`